server/xpub-model/entities/user/helpers/elife-api.js

```diff
@@ -56,4 +83,5 @@ module.exports = {
   person,
   peopleById,
   profile,
+  getEditorsByPersonId,
```

Comment:
This function should be removed from the `User` model if we're expecting to call it from the API instead

Reply:
In fact, it may be better off exposed as a static method under the `Team` model.